store/mongo/images_test.go

```diff
@@ -267,3 +267,45 @@ func TestIsArtifactUnique(t *testing.T) {
 	}
 
 }
+
+func TestArtifactUpdate(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping TestIsArtifactUnique in short mode.")
```

Comment:
small typo there ;) `TestArtifactUpdate`

Reply:
Ops, you are right; ready to force-push as soon as I get the test results.